### ISSUE TITLE
Portable version?

### ISSUE THREAD

**Issue by gbdematos** [bayleeadamoss/zazu#116]
Is it possible to make a portable version?

It doesn't allow to install without admin rights. Any reason for this?

I've tried extracting the installer and I have a folder with the contents, but running "Zazu.exe" does nothing.

**Comment by bayleedev** [comment]
I believe it needs admin rights because I haven't signed it. I'll be updating this in the future.

What is a portable version?

**Comment by gbdematos** [comment]
Portable is an application that you download, extract and run, without installing... Most of them doesn't require admin rights. I think it's called "portable" because people use to run them from USB sticks.

Thanks for the answer, I look forward into testing it! I'll try at home (where I have admin access) later.

**Comment by bayleedev** [comment]
Yes, I think the new packaging system will create those, when that gets released I'll ping you.

**Comment by schmic** [comment]
Portable means as well, that the configuration is kept inside the application folder and not in some $HOME-subfolder.

**Comment by tooomm** [comment]
I'm in for a portable version as well! 👍

**Comment by schmic** [comment]
@blainesch I made a quick change to the app/lib/configuration.js file

```
    const cwd = process.cwd()
    const portableHome = path.join(cwd, 'portable')
    if (require('fs').existsSync(portableHome)) {
      process.env.ZAZU_HOME = portableHome
    }
```

I think the code is pretty straight forward.
Should I throw that into a PR?

What it has as a requirement would be a simple .zip-file instead of an installation.

**Comment by bayleedev** [comment]
@schmic I'm working on a branch now that will output a zip file. How would you recommend building  two versions of the app? Pr's totallys welcome [:

**Comment by schmic** [comment]
@blainesch For packaging I'd use [electron-packager](https://github.com/electron-userland/electron-packager)

**Comment by bayleedev** [comment]
This was released in v0.4.0
